### ISSUE TITLE
IDEATION: Admin tools for when ideas are combined or duplicates 

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1299]
@kevincrepin @ilmartyrk

**Insights from admin and participant Round 1 ideation UX tests**

- Sometimes people add ideas that are two in one. This wasn't mentioned by any users, but my observation is this might make it difficult when it comes to voting. 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/79a33c53-95ab-4bdb-a001-79ade53c0f9f)

- Sometimes two users add ideas which are similar. I tested this in two participants tests (having two ideas that were similar), and both users noticed this and were looking for/asking about a way for either participants or admins to deal with this. 

One of the participants reported it to admin 'as a duplicate' but wasn't sure this felt the best course of action. It wasn't fully clear, but he seemed to be suggesting a way to message the user who had written the duplicate, or an option in the menu here to 'Report as duplicate' (he described as 'Nudging the person who created it'):  

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/1d326af7-b61c-4a87-b168-3377306de5af)

The other participant didn't try to do anything herself, but was asking if there was something the admin could do when there were duplicates, as she said that often happens in their processes. 

**Suggestions:**

Re combined ideas:
- Maybe the admin could have a way to message the creator of the idea, and ask them to separate out the idea into 2 before voting?
- Maybe there's a way they can edit the title in the vote - or not include the idea, but include 2 separate headings additionally? I.e. not include 'Trees and benches', but add manually 'Trees' and 'Benches'.  

Re duplicates:
-  This may just be something we leave and keep an eye on. Maybe the report to admin and the folder system are enough for this. Bur 'reporting' seems a bit extreme for duplication, when it's in a dropdown along with hate speech etc., and the person would receive a 'Reported as inappropriate' email! Maybe these two actions can be separated out?

**Comment by BeccaMelhuish** [comment]
@ilmartyrk @kevincrepin Up to you if this is something we can do for Round 2 or leave for later, not sure :)

**Comment by kevincrepin** [comment]
I kinda feel like this is why the folder system was added. If different people share same ideas, they can all be added to one folder. In discussion people can also come with duplicate arguments but we don't consider there to have the possibility to report this. I see these as being quite similar where if a similar idea keeps popping up, it might be an idea that would be popular also.

**Comment by BeccaMelhuish** [comment]
Yes this makes sense! (Though actually we do have 'Is a duplicate' in the reporting for arguments). I hadn't marked [this issue](https://github.com/citizenos/citizenos-fe/issues/1281 ) as a priority one, but what do you think about making the folders combine with a kind of tag functionality - so the names of the folders appear like tags on the idea, and participants can add them to folders themselves (to help the admin with the management of it)? Something probably to come back to later though, rather than to try to fix for Round 2.

**Comment by kevincrepin** [comment]
This (folder visible on idea) is the same issue as this one, right? https://github.com/citizenos/citizenos-fe/issues/1281#issuecomment-2158687666

**Comment by BeccaMelhuish** [comment]
Yes just ignore that now, thank you! :)
